core/src/main/java/org/microshed/testing/jwt/JwtConfigExtension.java

```diff
@@ -0,0 +1,97 @@
+package org.microshed.testing.jwt;
+
+import org.junit.jupiter.api.extension.*;
+import org.microshed.testing.internal.InternalLogger;
+import org.microshed.testing.jupiter.MicroShedTestExtension;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class JwtConfigExtension implements BeforeTestExecutionCallback, AfterTestExecutionCallback {
+
+    private static final InternalLogger LOG = InternalLogger.get(JwtConfigExtension.class);
+
+    @Override
+    public void beforeTestExecution(ExtensionContext context) throws Exception {
+        configureJwt(context);
+    }
+
+    @Override
+    public void afterTestExecution(ExtensionContext context) throws Exception {
+        removeJwt(context);
+    }
+
+    private void configureJwt(ExtensionContext context) throws Exception {
+
+        // Check if the test method has the @JwtConfig annotation
+        Method testMethod = context.getTestMethod().orElse(null);
+        if (testMethod != null) {
+
+            // Check if RestAssured is being used
+            Class<?> restAssuredClass = tryLoad("io.restassured.RestAssured");
+            if (restAssuredClass == null) {
+                LOG.debug("RESTAssured not found!");
+                return;
+            }
+
+            LOG.debug("RESTAssured found!");
+
+            JwtConfig jwtConfig = testMethod.getAnnotation(JwtConfig.class);
+            if (jwtConfig != null) {
```

Comment:
Check if annotation is present before attempting to load to the RestAssured class.